game/views.py

```
@@ -1276,3 +1277,26 @@ def confirm_delete_account(request, uidb64, token):
+        moves = data.get('moves', [])
+        result = data.get('result', 'Unknown')
+        reason = data.get('reason', 'Unknown')
+        
+        # Ensure moves is a list of strings
+        if not isinstance(moves, list):
+            moves = []
+        moves = [str(m) for m in moves]
+            
+        summary = build_summary(moves, result, reason)
+        return JsonResponse(summary)
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)